File: src/targets/python/requests/fixtures/jsonObj-multiline.py
import requests

<<<<<<< HEAD
url = "https://httpbin.org/anything"

payload = {"foo": "bar"}
=======
url = "http://mockbin.com/har"

payload = { "foo": "bar" }
>>>>>>> upstream/master
headers = {"content-type": "application/json"}

response = requests.post(url, json=payload, headers=headers)

<<<<<<< HEAD
print(response.text)
=======
print(response.json())
>>>>>>> upstream/master

File: src/targets/javascript/jquery/fixtures/jsonObj-multiline.js
const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/anything',
=======
  url: 'http://mockbin.com/har',
>>>>>>> upstream/master
  method: 'POST',
  headers: {
    'content-type': 'application/json'
  },
  processData: false,
  data: '{\n  "foo": "bar"\n}'
};

$.ajax(settings).done(function (response) {
  console.log(response);
});